release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -109,14 +112,72 @@ With granular update control, you can select a specific Operator version or defi
 +
 Alternatively, automatic z-stream updates provide a seamless and secure experience by automatically applying security fixes without manual intervention, minimizing operational disruptions.
 
-Enhanced security with user-provided `ServiceAccount` objects::
-{olmv1} prioritizes security by minimizing its permission requirements and providing greater control over access. By using user-provided `ServiceAccount` objects for Operator lifecycle operations, {olmv1} access is restricted to only the necessary permissions, significantly reducing the control plane attack surface and improving overall security. In this way, {olmv1} adopts a least-privilege model to minimize the impact of a compromise.
+Enhanced security with user-provided service accounts::
+{olmv1} prioritizes security by minimizing its permission requirements and providing greater control over access. By using xref:../extensions/ce/managing-ce.adoc#olmv1-cluster-extension-permissions_managing-ce[user-provided `ServiceAccount` objects for Operator lifecycle operations], {olmv1} access is restricted to only the necessary permissions, significantly reducing the control plane attack surface and improving overall security. In this way, {olmv1} adopts a least-privilege model to minimize the impact of a compromise.
+
+[NOTE]
+====
+The documentation for {olmv1} exists as a stand-alone guide called xref:../extensions/index.adoc#olmv1-about[Extensions]. Previously, {olmv1} documentation was a subsection of the xref:../operators/index.adoc#operators-overview[Operators] guide, which otherwise documents the {olmv0} feature set.
+
+The updated location and guide name reflect a more focused documentation experience and aims to differentiate between {olmv1} and {olmv0}.
+====
 
 [id="ocp-4-18-extensions-supported-extensions_{context}"]
-==== {olmv1} supported extensions and known issue
+==== {olmv1} supported extensions
 
 include::snippets/olmv1-tp-extension-support.adoc[]
 
+[id="ocp-4-18-extensions-disconnected_{context}"]
+==== Disconnected environment support in {olmv1}
+
+To support cluster administrators that prioritize high security by running their clusters in internet-disconnected environments, especially for mission-critical production workloads, {olmv1} includes cluster extension lifecycle management functionality that works within these disconnected environments, starting in {product-title} {product-version}.
```

Comment:
We should just say `{olmv1} supports these disconnected environments, starting in {product-title} {product-version}.`